distribution/docker/ironbank-docker-build-context/build.gradle

```diff
@@ -0,0 +1,14 @@
+import org.elasticsearch.gradle.Architecture
+import org.elasticsearch.gradle.DockerBase
+
+apply plugin: 'base'
+
+tasks.register("buildIronBankDockerBuildContext", Tar) {
+  archiveExtension = 'tar.gz'
+  compression = Compression.GZIP
+  archiveClassifier = "docker-build-context"
+  archiveBaseName = "elasticsearch-ironbank"
+  with dockerBuildContext(Architecture.X64, false, DockerBase.IRON_BANK, false)
+}
+
+tasks.named("assemble").configure {dependsOn "buildIronBankDockerBuildContext"}
```

Comment:
Should we be wiring this up to the `assemble` task if we generally don't want to be building this image as part of a "normal" build? Since building this for now is going to be a manual process perhaps we should leave this bit out and just run the explicit task when needed.

Reply:
That's a fair point. I removed the wiring-up.